gender-yearsExperience.sh

```diff
@@ -0,0 +1,5 @@
+#sort wages.csv by gender then years experience
+#usage bash gender-yearsExperience
+
+cat wages.csv | cut -f 1,2 -d "," | sed 's/,/ /g' | sort -k 1,1d -k 2,2n
```

Comment:
Please read the questions carefully before finishing. Missing uniq at the end of the pipeline. Results should be written to a file. 
-0.5 pts

Reply:
When provided, comments in code are descriptive. Please comment code consistently across script. 
Work on providing more descriptive commit messages for collaborators and future you :)